### PR TITLE
add MyMakers RP2040 board

### PR DESCRIPTION
**Pull request by nui4328**
The MyMakers RP2040 board is developed for use in studying the creation and programming of small robot controllers. for students It is distributed in public places.

### CODE REVIEW

File: tools/makeboards.py [hunk 1]
```diff
@@ -684,6 +684,9 @@ def MakeBoardJSON(name, chip, vendor_name, product_name, vid, pid, pwr, boarddef
 MakeBoard("generic", "rp2040", "Generic", "RP2040", "0x2e8a", "0xf00a", 250, "GENERIC_RP2040", 16, 0, "boot2_generic_03h_4_padded_checksum")
 MakeBoard("generic_rp2350", "rp2350", "Generic", "RP2350", "0x2e8a", "0xf00f", 250, "GENERIC_RP2350", 16, 8, "none")
 
+# MyMakers
```
Comment (earlephilhower) [inline tools/makeboards.py:687]: Almost there, but we keep things alphabetized in the menus.  Please move this to after the Olimex board and re-run and commit.  OTW, looks good.

**Review by earlephilhower — CHANGES_REQUESTED** [review]

File: tools/makeboards.py (hunk 1 above)
Comment (nui4328) [reply tools/makeboards.py:687]: # Olimex
MakeBoard("olimex_rp2040pico30_2mb", "rp2040", "Olimex", "RP2040-Pico30 2MB", "0x15ba", "0x0026", 250, "OLIMEX_RP2040_PICO30_2MB", 2, 0, "boot2_w25q080_2_padded_checksum")
MakeBoard("olimex_rp2040pico30_16mb", "rp2040", "Olimex", "RP2040-Pico30 16MB", "0x15ba", "0x0026", 250, "OLIMEX_RP2040_PICO30_16MB", 16, 0, "boot2_w25q080_2_padded_checksum")

# MyMakers
MakeBoard("MyRP_bot", "rp2040", "MyMakers", "RP2040", "0x2e8a", "0x000a", 250, "MyRP_2040", 2, 0, "boot2_generic_03h_4_padded_checksum")

Is this correct?

**Review by earlephilhower — APPROVED** [review]
It took some time but looks like we got there in the end!

**Comment by nui4328** [comment]
Thank you so much

เมื่อ อ. 17 ธ.ค. 2567 เวลา 10:13 Earle F. Philhower, III <
***@***.***> เขียนว่า:

**Pull request merged**